corim-code-points.cddl

```diff
@@ -0,0 +1,17 @@
+; unsigned-corim
+corim-id = 123 ; TBD
+tags = 97
+dependent-rims = 122; TBD
+
+; corim-meta
+signer = 98
+deny-id = 99
```

Comment:
I think we should have a `corim.` prefix for these?

Reply:
These are existing code points moved here, so this can be done as part of a file wide clean up?

Reply:
sure, I was suggesting we could do it now since we are moving them around alread

Reply:
Raised Issue #22 to address this comment!